apps/landing/src/pages/index.tsx

```diff
@@ -98,9 +98,9 @@ function Page() {
 				</Button>
 			</div>
 			<p className="px-6 mt-3 text-sm text-center opacity-75 text-gray-450">
-				Coming soon on macOS, Windows and Linux.
+				Coming soon to macOS, Windows, and Linux.
 				<br />
-				Shortly after to iOS & Android.
+				Then shortly after to iOS & Android.
```

Comment:
This actually makes _less_ grammatical sense to me than the previous version.

```suggestion
				Shortly after to iOS & Android.
```

If that original wording really was unclear, you could write this instead:

```suggestion
				Shortly thereafter to iOS & Android.
```